src/pybind/mgr/mgr_module.py

```diff
@@ -351,6 +351,19 @@ def CLIWriteCommand(prefix, args="", desc=""):
     return CLICommand(prefix, args, desc, "w")
 
 
+def CLICheckNonemptyFileInput(func):
+    def check(*args, **kwargs):
+        if not 'inbuf' in kwargs:
+            return -errno.EINVAL, '', \
+                   'Please specify the file containing the password/secret with "-i" option.'
+        if not kwargs['inbuf'] or (isinstance(kwargs['inbuf'], str)
+                                   and not kwargs['inbuf'].strip('\n')):
+            return -errno.EINVAL, '', \
+                   'The file containing the password/secret CANNOT be empty.'
```

Comment:
Are there enough characters available in this error message to include something like "Add a password/secret"? This is minor, but it tells the user with the empty file what to do next.